using-kubecost/admission-controller.md

```diff
@@ -0,0 +1,50 @@
+# Admission Controller
+
+{% hint style="info" %} The Admission Controller is a beta feature. Please read the documentation carefully. {% endhint %}
+
+Kubecost's Admission Controller is a `kubectl` integration which leverages Kubecost's Predict API to display expected future costs for changes made to CPU and RAM in your workloads. This can be used to anticipate future spend before resources have been allocated.
+
+The Admission Controller performs using a `ValidatingAdmissionWebhook` to send back cost data on every update you make to deployments in your Kubernetes clusters. The Predict API then determines the differences in cost, and provides that information in a convenient table in your terminal.
+
+## Installing the Admission Controller
+
+### Prerequisites
+
+Before installing the Admission Controller, make sure you have [`kubectl`](https://kubernetes.io/docs/reference/kubectl/) installed.
+
+### Installation
+
+The following command will install the Admission Controller to your existing Kubecost application:
+
+```
+helm upgrade --install kubecost \
+  --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer \
+  --namespace kubecost --create-namespace \
+  --set kubecostAdmissionController.enabled=true
+```
+
+You may need to wait several minutes for the controller to activate. You can check the status of the Admission Controller with `kubectl get service -n kubecost`, when the Admission Controller has been installed in the default `kubecost` namespace. Look for `webhook-server` to confirm a successfull install.
+
+## Using the Admission Controller
+
+When a deployment is updated, the Kubernetes API will send requests containing deployment information to the Kubecost pod, which will then be read for number of replicas, CPU requests, and RAM requests to calculate a monthly estimate. That estimate will be reported back to the client making the update. If the Kubecost pod is unable to respond to this request, the deployment will be added to the cluster without any information being sent back.
```

Comment:
```suggestion
When a Deployment is updated, the Kubernetes API will send requests containing Deployment information to the Kubecost pod, which will then be read for number of replicas, CPU requests, and RAM requests to calculate a monthly cost estimate. That estimate will be reported back to the client making the update. If the Kubecost pod is unable to respond to this request, the deployment will be added to the cluster without any information being sent back.
```

This is an excellent summary.

Reply:
Thanks, but this is loosely taken from our blog post, so I'm pretty sure you wrote this